cloudbuild/python-scripts/parse_logs.py

```diff
@@ -0,0 +1,281 @@
+"""Python Script to parse yarn logs to check of job success.
+
+This python file extracts the status of a dataproc job from its job-id and then
+uses this job-id to get the yarn application number.
+The yarn application number enables it to read through the yarn logs to check
+for the number iof records.
+In case the number of records match that of BQ table it returns, in case of
+mismatch, it throws an error.
+"""
+
+from collections.abc import Sequence
+
+from absl import app
+from google.cloud import bigquery
+from google.cloud import dataproc_v1
+from google.cloud import storage
+import re
+
+
+def get_bq_query_rows(client_project_name):
```

Comment:
Should be renamed to `get_bq_query_result_row_count`

Reply:
Done.